crates/compilers/src/flatten.rs

```diff
@@ -904,3 +899,12 @@ pub fn combine_version_pragmas(pragmas: Vec<&str>) -> Option<String> {
 
     None
 }
+
+pub fn apply_updates(sources: &mut Sources, mut updates: Updates) {
```

Comment:
not necessary w/ other comments

```suggestion
pub fn apply_updates(sources: &mut Sources, updates: &Updates) {
```

Reply:
applied in same https://github.com/foundry-rs/compilers/pull/252/commits/d37cafb8f47521c4e50b126db2acac6dcd4e002a